src/doc/gui_manual/Plots/PlotTypes/MoleculePlot.rst

```diff
@@ -0,0 +1,140 @@
+number. Also, note that some default values are set due to much molecular data 
+being in units of Angstroms. Depending on your data, you may need to change the 
+atomic/bond radii.
+
+When **Radius based on** is set to *Scalar variable*, the 
+**Variable for atom radius** field becomes active and determines which variable 
+shall be used (and multiplied by the scale factor below) as the value for the 
+radius of the rendered atoms.
+
+**Atom radius scale factor** applies when **Radius based on** is not set to 
+*Fixed value*. This value multiplies the other value used for radius, whether 
+it is the atomic/covalent radius or based on a scalar variable. Note that the 
+atomic and covalent radii used are in Angstroms, so if your data is in other 
+units, you should apply the appropriate conversion factor here.
+
+**Fixed atom radius** only applies when **Radius based on** is set to 
+*Fixed value*. It is the actual radius you want to use to draw the atoms in 
+world coordinate units. 
+
+
+Controlling how bonds are drawn
+"""""""""""""""""""""""""""""""
+
+.. _molecule_bonds:
+
+.. figure:: ../images/molecule_bonds.png
+
+   Molecule plot window Bonds tab
+
+The value *None* for **Draw bonds as** means you are only interested in seeing 
+the atoms and would like any bonds to be hidden. *Lines* uses geometric lines 
+with no 3D shading, and *Cylinders* uses 3D geometry with 3D shading. 
+*Lines* is much faster but *Cylinders* looks better.
+
+When **Draw bonds as** is set to *Cylinders*, **Bond cylinder quality** 
+determines the number of polygons used to draw the bonds. *Low* is about three 
+polygons, and *High* is about twenty.
+
+When **Draw bonds as** is set to *Cylinders*, **Bond radius** determines the 
+thickness of the cylinder in world coordinate units. Note that defaults for 
+these values were chosen due to molecular data commonly being in units of 
+Angstroms. Depending on your data, you may need to change the radius used 
+for rendering atoms and bonds.
+
+When **Draw bonds as** is set to *Lines*, **Bond line width** determines the 
+thickness of the line used to draw the bonds in terms of a number of pixels.
+
+**Color bonds by** can be set to *Adjacent atom color*, which means that each 
+half of the bond is drawn using the color of the atom to which it is attached. 
+Or, it can be set to a *Single color* chosen at the color selector just to the 
+right of this checkbox. 
+
+
+Controlling colors 
+""""""""""""""""""
+
+.. _molecule_colors:
+
+.. figure:: ../images/molecule_colors.png
+
+   Molecule plot window Colors tab
+
+
+The **Discrete colors group** is for values which take on integral values. 
+When VisIt_ encounters a discrete-valued variable, it determines which one of 
+these color tables to use based on the variable name (*element* and *restype*, 
+specifically).
+
+**Element types** and **Residue types** are specific examples, and are separate 
```

Comment:
One small grammar error here:
1. "...are specific examples, and are separate...". This should either have its comma removed or be changed to "...are specific examples, and they are separate..."

Reply:
done.